editoast/src/client/mod.rs

```diff
@@ -50,6 +50,22 @@ pub enum Commands {
     Search(SearchCommands),
     #[command(subcommand, about, long_about = "Infrastructure related commands")]
     Infra(InfraCommands),
+    #[command(subcommand, about, long_about = "Trains related commands")]
+    Trains(TrainsCommands),
+}
+
+#[derive(Subcommand, Debug)]
+pub enum TrainsCommands {
+    Import(ImportTrainArgs),
+}
+
+#[derive(Args, Debug, Derivative)]
+#[derivative(Default)]
+#[command(about, long_about = "Import a train given a file")]
+pub struct ImportTrainArgs {
+    #[arg(long)]
+    pub timetable: Option<String>,
```

Comment:
We can let clap parse the number.

```suggestion
    pub timetable: Option<u64>,
```